wagtail/admin/templates/wagtailadmin/shared/shared_avatar.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags %}
```

Comment:
This file does not need to be `shared_avatar.html` as it is already in a `shared` folder.

Reply:
i was unsure to user `avatar.html` as it already exists